src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -1325,16 +1325,10 @@ class misc_commandscript : public CommandScript
 
     static bool HandleMaxSkillCommand(ChatHandler* handler, char const* /*args*/)
     {
-        Player* SelectedPlayer = handler->getSelectedPlayer();
```

Comment:
Might be wise to keep the null check. I'm not sure if getSelectedPlayerOrSelf() never returns null (perhaps when calling the command from the console?)